app/models/enrollment.rb

```diff
@@ -40,6 +40,13 @@ class Enrollment < ApplicationRecord
       Enrollment::SendMailJob.perform_now(enrollment, user, event)
     end
 
+    after_transition :pending => :sent do |enrollment, transition|
```

Comment:
when testing your code locally, I do not see this executed ? Can you show me how to make it works ?